exporter/api.py

```diff
@@ -37,8 +38,30 @@ def sanitize(title):
                 id=learning_resource.id,
                 title=sanitize(learning_resource.title),
             )
-            with open(os.path.join(tempdir, filename), 'w') as f:
+            type_name = learning_resource.learning_resource_type.name
+            try:
+                os.mkdir(os.path.join(tempdir, type_name))
+            except OSError:
+                pass
```

Comment:
Both ways would work, I'm going to leave it the way it is since the change was made in the other PR (this PR includes bits of the other PR)